dashboard/test/models/stage_test.rb

```diff
@@ -133,6 +133,101 @@ class StageTest < ActiveSupport::TestCase
     assert_equal '/', stage2.next_level_path_for_stage_extras(@student)
   end
 
+  test "published? returns true if levelbuilder" do
+    Timecop.freeze(Time.new(2020, 3, 27))
```

Comment:
when a bunch of your tests use the same Timecop command, you can move those to a `setup` and `teardown` method so that you don't have to call them in each test case.  if you don't want these to apply to all of your tests, you can put them inside a nested class, like this: https://github.com/code-dot-org/code-dot-org/blob/151af43fd62aca99b1bee003f3ab2bb85f846851/dashboard/test/models/course_test.rb#L3-L6